modules/tournament/src/main/crud/CrudForm.scala

```diff
@@ -22,7 +23,7 @@ object CrudForm {
     "clockTime" -> numberInDouble(clockTimePrivateChoices),
     "clockIncrement" -> numberIn(clockIncrementPrivateChoices),
     "minutes" -> number(min = 20, max = 1440),
-    "variant" -> number.verifying(v => variantFromId(v).isDefined),
+    "variant" -> number.verifying(v => Variant.byId.get(v).isDefined),
```

Comment:
I'm really nitpicking here, but since `Variant.byId` is a `Map[Int, Variant]`, then `number.verifying(Variant.byId.contains)` could also work. Just FYI, no need to change it.